user/caching.md

```diff
@@ -135,7 +135,16 @@ podfile: path/to/Podfile
 
 ### npm cache
 
-For caching with `npm`, use:
+> Please note that npm is now cached by default on Travis CI
```

Comment:
Can we say as of Xenial, or as of July 2019 or something more specific?

Reply:
yes, we can totally say `as of July 2019`